src/components/navigation/Link.js

```diff
@@ -0,0 +1,56 @@
+import classnames from 'classnames';
+
+import { downcastRef } from '../../util/typing';
+
+import LinkBase from './LinkBase';
+
+/**
+ * @typedef {import('../../types').PresentationalProps} CommonProps
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLAnchorElement>} HTMLAnchorAttributes
+ *
+ * @typedef LinkProps
+ * @prop {'always'|'hover'|'none'} [underline]
+ * @prop {'brand'|'text-light'|'text'} [color='brand']
+ */
+
+/**
+ * Styled component for a link (`<a>` element).
+ *
+ * @param {CommonProps & LinkProps & HTMLAnchorAttributes} props
+ */
+export default function Link({
+  children,
+  classes,
+  elementRef,
+
+  underline = 'none',
+  color = 'brand',
+
+  ...htmlAttributes
```

Comment:
Because it can become a world of hurt to have rest props that may include properties from multiple sources, I've established the convention that only HTML attributes should be rest props, and the other props expanded out, even if it's just to essentially forward them on. I've also ordered props by their source and put line breaks between sources. I find this easier to reason about...

Reply:
> I've established the convention that only HTML attributes should be rest props, and the other props expanded out, even if it's just to essentially forward them on.

Makes sense.